modules/output/__init__.py

```diff
@@ -88,13 +88,11 @@ def log_trades(stats: TradingStats):
                       'currency_amount': trade.currency_amount,
                       'sell_reason': trade.sell_reason}
 
-        trades_dict[i] = trade_dict
-
-    trades_dict = dict(sorted(trades_dict.items()))
+        trades_dict["trades"][i] = trade_dict
 
     trades_json = json.dumps(trades_dict, indent=4, default=str)
 
-    with open('./data/backtesting-data/trades_log.json', 'w', encoding='utf-8') as f:
+    with open(f'./data/backtesting-data/{config.strategy_name}_trades_log.json', 'w', encoding='utf-8') as f:
```

Comment:
I would change this to:

```suggestion
    with open(f'./data/backtesting-data/trades_log_{config.strategy_name}.json', 'w', encoding='utf-8') as f:
```

Because that makes it more clear from the get-go that it's a trades log. If you have multiple underneath each other it's clear that while the strategy differs, they're all trade logs.